README.md

```diff
@@ -1,63 +1,59 @@
 # MapleDiscBot
-"Universal" plug n play maplestory discord bot for your private server.
-Any questions or bug reports, my discord is Not Brandon#4444
+### "Universal" plug n play maplestory discord bot for your private server.
+If you have any questions or bug reports, my discord is **not** Brandon#4444
 
 I tried to make this maplestory discord bot as plug n play as possible but you are still going to need to some work.
-Note, I'm don't claim to be a good programmer and there might be some memes in here but it could help newer people
-and I think its time I released something :P
+Note, I don't claim to be a good programmer but I think this could help people who are new at creating a Discord bot and I think its time I released something for them :P
 
 Make sure to use Visual Studio Code for any debugging or editing.
 
 ## 1. How to get a discord bot token? <h2>
-1. discordapp.com
+1. Visit the [developer site](https://discord.com/developers/applications)
   
-2. Developers tab at the top then select developer portal
-![Discord Taskbar](https://media.discordapp.net/attachments/631249406775132182/722900574022139925/fa160fc47785eeb2a73f763244dcef34.png)
-
-3. Top right click new Application, name it accordingly
+2. Top right click new Application, name it accordingly
   
 ![Discord Application Button](https://cdn.discordapp.com/attachments/631249406775132182/722901729573863424/eb9cd2edd04845b4a5a9581f7f897cc1.png)
 
-4. Now click on your new application you just created
+3. Now click on your new application you just created
```

Comment:
```suggestion
3. Now click on the new application that you've just created
```

Reply:
I actually went through the process of creating a new bot to verify the sequence.
It automatically selects the new application without needing to "click on your new application you just created".
That's why I suggested the edit to ascertain that it's selected, rather than clicking on the application name from the drop-down menu (which wouldn't do anything)